platform/common/services/logging/otel.go

```diff
@@ -36,8 +35,7 @@ type otelLogger interface {
 func NewOtelLogger(zapLogger *zap.Logger) otelLogger {
 	return otelzap.New(zapLogger,
 		otelzap.WithLoggerProvider(newLoggerProvider(zapLogger.Name())),
-		//otelzap.WithMinLevel(zapLogger.Level()),
-		otelzap.WithMinLevel(zapcore.DebugLevel),
+		otelzap.WithMinLevel(zapLogger.Level()),
```

Comment:
we could use the already existing funciton   
```go
func Init(c Config) {
flogging.Init(c)
}
```
to allow a further customization for the otel level. So that even if we have info for the terminal, we have debug for otel.

Reply:
> Can we still add trace information to spans without also logging?

yes - you, you just need an instance of a span and then you create an event.

Example:
```
somespan.AddEvent("Read response")
```